### ISSUE TITLE
Make pending migration error a little more noticable

### ISSUE THREAD

**Issue by jwoertink** [luckyframework/avram#671]
With so much going on booting a Lucky app, it's easy to miss that there's some pending migrations. 

I think we can do better.

<img width="1449" alt="Screen Shot 2021-04-27 at 2 02 16 PM" src="https://user-images.githubusercontent.com/2391/116312821-d0b32600-a761-11eb-81e4-208dac1e7311.png">

**Comment by matthewmcgarvey** [comment]
Related https://github.com/luckyframework/lucky/issues/1142

**Comment by robacarp** [comment]
I think the information density on that startup screen is pretty confusing as well. If you're in a hurry you might not notice that the giant blue banner at the top isn't the URL you want to click, you want the little tiny one down below called "proxying" which comes from browsersync, and coupled with the sometimes double-nesting of log namespaces it's a bit overwhelming (`assets       | [browsersync]`). Maybe there's some logic we can add to the dev script that doesn't print the success banner when it's going to be served behind browsersync?

**Comment by jwoertink** [comment]
Yeah, it's definitely a lot of info. I thought I made an issue somewhere, but maybe it was on discord... I came across some repo someone wrote this amazing terminal UI all in Go. I saw it and said "That would be baller if lucky booted looking like that". I'll have to find it again...

Also on that note... This error looks like it's a runtime error... What if we treated it like compile-time, and just called `exit` after we raised that error? But still made the error message pretty... 🤔
